smoke-tests/spec/ingress_spec.rb

```diff
@@ -40,15 +40,15 @@
     end
   end
 
-  context "when ingress is deployed with invalid syntax" do
-    it "is rejected by the admission webhook" do
-      stdout_str, stderr_str, status = apply_template_file(
-        namespace: namespace,
-        host: host,
-        file: "spec/fixtures/invalid-nginx-syntax.yaml.erb",
-        binding: binding,
-      )
-      expect(stderr_str).to match(/admission webhook "validate.nginx.ingress.kubernetes.io" denied the request/)
-    end
-  end
+  # context "when ingress is deployed with invalid syntax" do
```

Comment:
@vijay-veeranki-moj you should be able to just change `context` to `xcontext` to achieve this.
That has the added benefit of reporting, in the test output, that the test is being skipped, so that we don't forget about it.

Reply:
Thanks @digitalronin, made the change , can you please review it again